src/main/java/pl/psnc/dei/model/Aggregator.java

```diff
@@ -13,11 +17,6 @@ public int getId() {
 		return id;
 	}
 
-	Aggregator(int id, String fullName) {
-		this.id = id;
-		this.fullName = fullName;
-	}
-
 	public String getFullName() {
```

Comment:
After adding @Getter annotation it would reasonable to remove the code for the getters.

Reply:
fixed